### PR TITLE
Add attribues in XML for increased cobertura compatibility

### PR DESCRIPTION
**Pull request by wahln**
Since I had some problems with parsing an coverage XML file exported with MOcov in a GitHub Action, this PR adds a few attributes to the top-level `coverage` node of the written XML file according to the [cobertura format definition](https://github.com/cobertura/web/blob/master/htdocs/xml/coverage-04.dtd). These attributes are:
- lines-covered
- lines-valid
- branches-covered (dummy value of 0)
- branches-valid (dummy value of 0)
- complexity (dummy value of 0)

lines-covered and lines-valid are computed within the `get_coverage` function within `write_xml_file` anyway (`numerator` and `denominator`), so I just added those values as additional output of said function and feed them into the coverage node.
The other values are not available, so I set them to zero.

There's also a function `write_cobertura_xml`, but it seems not to be used, so I didn't change it for now. Let me know if this function should contain the changes as well.

### CODE REVIEW

File: MOcov/@MOcovMFileCollection/write_xml_file.m [hunk 1]
```diff
@@ -17,11 +17,11 @@ function write_xml_file(obj, output_fn)
     notify(monitor,sprintf('Writing xml files in %s', output_fn));
 
     % write header
-    overall_coverage=compute_coverage(obj);
+    [overall_coverage,lines_covered,lines_valid]=compute_coverage(obj);
     notify(monitor,sprintf('Overall coverage is %.3f', overall_coverage));
     header=sprintf(['<?xml version="1.0"?>\n'...
-                    '<coverage line-rate="%.3f" branch-rate="%.3f">'],...
-                    overall_coverage,branch_rate);
+                    '<coverage line-rate="%.3f" branch-rate="%.3f" lines-covered="%d" lines-valid="%d" branches-covered="0" branches-valid="0" complexity="0">'],...
```
Comment (nno) [inline MOcov/@MOcovMFileCollection/write_xml_file.m]: Could you please limit the line length here as done elsewhere in the code, by splitting up this line into multiple ones and use line continuation?

**Comment by nno** [comment]
Thanks, this seems like a useful addition. Just a minor request regarding line length. After that I would be happy to merge this into the master branch.

File: MOcov/@MOcovMFileCollection/write_xml_file.m (hunk 1 above)
Comment (wahln) [reply MOcov/@MOcovMFileCollection/write_xml_file.m]: Sure, pushed the change in the newest commit.

Comment (nno) [reply MOcov/@MOcovMFileCollection/write_xml_file.m]: Thanks!

**Pull request merged**